add first sql schema draft

Merovingian87 opened the pull request.

Merovingian87 committed:
add first sql schema draft

phuctranlv commented on db/schema.sql:
```diff
@@ -0,0 +1,42 @@
+ALTER TABLE comments ADD CONSTRAINT constraint_fk2 FOREIGN KEY (user_id)
+REFERENCES users(id);
+
+ALTER TABLE replies ADD CONSTRAINT constraint_fk FOREIGN KEY (comment_id)
```
shouldn't there a reference to a user too for replies?

Merovingian87 replied:
100%, thanks! I will update and commit

phuctranlv reviewed (changes requested):
Please see comment regarding replies reference constraint

Merovingian87 committed:
Finish schema.sql file for postgreSQL

I finished testing my schema.sql with PSQL, it workes and created the
proper forign keys for my queries.
